src/main/java/com/example/model/Book.java

```diff
@@ -4,15 +4,13 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import java.math.BigDecimal;
-import lombok.Getter;
-import lombok.Setter;
-import lombok.ToString;
+import lombok.Data;
 
+@Data

```

Comment:
revert those changes, @ Data contains problematic annotations for JPA entities: toString() (when related entities are lazy initalized), equalsAndHashcode(), so use only the things you need, or explicitly deal with them like using @equalsandhashcode.exclude etc